### PR TITLE
Fix bug #563

### PR DESCRIPTION
**Pull request by ericdevg**
#563

Fix: Activate window before execute command.

@AndreSayreMSFT @HuanhuanSunMSFT @MrayerMSFT @Microsoft/poshtools

### CODE REVIEW

File: PowerShellTools.HostService/MainWindow.xaml.cs [hunk 1]
```diff
@@ -26,9 +26,14 @@ public MainWindow()
...
-            this.Visibility = System.Windows.Visibility.Hidden;
-            this.Topmost = true;
+            this.Visibility = System.Windows.Visibility.Visible;
             this.ShowInTaskbar = false;
+            this.Title = string.Format(
+                    PowerShellTools.Common.Constants.HostProcessWindowTitleFormat, 
+                    App.VsProcessId, 
+                    PowerShellTools.Common.Constants.PowerShellHostExeName);
+            this.Left = -1000;
+            this.Top = -1000;
```
Comment (ASayre) [inline PowerShellTools.HostService/MainWindow.xaml.cs]: VSTO had to make a similar workaround once upon a time.   Guidance from the Windows team had some other specific constants to use.  Will dig up that information for you.

File: PowerShellTools.HostService/MainWindow.xaml.cs [hunk 2]
```diff
@@ -26,9 +26,14 @@ public MainWindow()
         {
             InitializeComponent();
 
-            this.Visibility = System.Windows.Visibility.Hidden;
-            this.Topmost = true;
+            this.Visibility = System.Windows.Visibility.Visible;
```
Comment (ASayre) [inline PowerShellTools.HostService/MainWindow.xaml.cs]: Should we only set this after we set the taskbar/title/left/top?

File: PowerShellTools.HostService/MainWindow.xaml.cs (hunk 1 above)
Comment (ericdevg) [inline PowerShellTools.HostService/MainWindow.xaml.cs]: Fixed.

**Comment by ASayre** [comment]
Looks good!

**Pull request merged**